Review-site.drawio

```diff
@@ -1,41 +1,41 @@
-<mxfile host="app.diagrams.net" modified="2020-10-19T19:21:26.450Z" agent="5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/86.0.4240.80 Safari/537.36" etag="8TgZSFYhmG7Q6xmb02zj" version="13.7.9" type="github">
```

Comment:
Are these 3 Draw.io files being used in the application?

Reply:
initially when we created the draw.io file it needed to be saved somewhere I can remove these if its a bummer